### PR TITLE
Issue 4 Deleting an Entry

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-02-11 at 4 16 22 PM
Screen Shot 2021-02-11 at 4 16 29 PM
Screen Shot 2021-02-11 at 4 25 30 PM
Screen Shot 2021-02-11 at 4 25 40 PM

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -41,16 +43,16 @@ $form.addEventListener('submit', function (event) {
...
   data.nextEntryId++;
   resetForm();
+  showEntries();
 });
 
 $newButton.addEventListener('click', function (event) {
-  $containerElements[0].className = 'container';
-  $containerElements[1].className = 'container hidden';
+  showNewEntry();
+  $delete.className = 'delete invisible';
```
Comment (codingcodymiller) [inline js/main.js]: Why not just use `hidden`?

File: js/main.js [hunk 2]
```diff
@@ -126,3 +130,52 @@ function editEntry(newData, DOMElement) {
...
+  $confirmationModal.setAttribute('class', 'confirmation-modal');
+  var $modal = $confirmationModal.appendChild(document.createElement('div'));
+  $modal.setAttribute('class', 'modal');
+  var $modalText = $modal.appendChild(document.createElement('h3'));
+  $modalText.textContent = 'Are you sure you want to delete this entry?';
+  var $cancelButton = $modal.appendChild(document.createElement('button'));
+  $cancelButton.setAttribute('class', 'cancel-button');
+  $cancelButton.textContent = 'cancel';
+  var $deleteButton = $modal.appendChild(document.createElement('button'));
+  $deleteButton.setAttribute('class', 'delete-button');
```
Comment (codingcodymiller) [inline js/main.js]: This does not need to be dynamically created, you can write the delete modal directly in your HTML file, and start it out as hidden.

File: css/layout.css [hunk 3]
```diff
@@ -2,6 +2,10 @@
   display: none !important;
 }
 
+.invisible {
+  visibility: hidden;
+}
+
```
Comment (codingcodymiller) [inline css/layout.css]: This is likely not necessary, since you already have a `hidden` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳 🥳

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-11 at 6 44 20 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**